azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/constants.py

```diff
@@ -0,0 +1,70 @@
+from os import environ
+from pathlib import Path
+from sys import platform
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+_LOG_PATH_LINUX = "/var/log/applicationinsights"
+_LOG_PATH_WINDOWS = "\\LogFiles\\ApplicationInsights"
+# TODO: Enabled when duplciate logging issue is solved
+# __EXPORTER_DIAGNOSTICS_ENABLED_ENV_VAR = (
+#     "AZURE_MONITOR_OPENTELEMETRY_DISTRO_ENABLE_EXPORTER_DIAGNOSTICS"
+# )
+
+
+def _get_log_path(status_log_path=False):
+    if platform.startswith("linux"):
+        return _LOG_PATH_LINUX
+    elif platform.startswith("win32"):
+        log_path = str(Path.home()) + _LOG_PATH_WINDOWS
+        if status_log_path:
+            return log_path + "\\status"
+        else:
+            return log_path
+    else:
+        return _LOG_PATH_LINUX
```

Comment:
Maybe we should disable the feature if the OS is unrecognized/unsupported?

Reply:
I was thinking of defaulting to Linux since I think there are some edge cases like Cygwin. But I think you're right. Will disable.

Reply:
done